target-config.js

```diff
@@ -119,7 +119,8 @@
       webAnimations1BonusSrc: webAnimations1BonusSrc,
       webAnimationsNextSrc: [],
       src: scopeSrc.concat(sharedSrc).concat(webAnimations1Src).concat(webAnimations1BonusSrc),
-      test: webAnimations1Test,
+      polyfillTests: webAnimations1PolyfillTests,
+      runWebPlatformTests: false,
```

Comment:
Hmm, this should probably be a list of files that we think native Chrome should pass instead, will take a look.
It's lame that webAnimations1 actually means a subset of level 1, if the name were accurate then we'd by trying to make it pass all the tests.